docs-2.0/nebula-explorer/deploy-connect/ex-ug-deploy.md

```diff
@@ -1,6 +1,17 @@
 # Deploy Explorer
 
-This topic describes how to deploy Explorer locally by RPM, and tar package.
+This topic describes how to deploy Explorer locally by RPM and tar package.
+
+## Nebula Graph version
+
+!!! Note
+
+    Explorer is released separately, not synchronized with Nebula Graph core. And the naming of Explorer is different from that of Nebula Graph. The version correspondence between Nebula Graph and Explorer is as follows.
```

Comment:
naming of Explorer is different from that of Nebula Graph?
That is the fact... 
You mean version naming?